org.catacombing.kumo.metainfo.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2025 Christian Duerr <contact@christianduerr.com> -->
+<component type="desktop-application">
+    <id>org.catacombing.kumo</id>
+    <metadata_license>CC0-1.0</metadata_license>
+    <project_license>GPL-3.0-only</project_license>
+    <name>Kumo</name>
+    <summary>A Wayland Mobile Browser</summary>
+    <developer id="org.catacombing">
+        <name>Christian Duerr</name>
+    </developer>
+    <description>
+        <p>
+            Kumo is a web browser with a UI focused on portrait-mode touchscreen
+            mobile devices. It is optimized to run on low-end hardware with a
+            limited battery life.
+        </p>
+    </description>
+    <launchable type="desktop-id">org.catacombing.kumo.desktop</launchable>
+    <provides>
+        <id>Kumo.desktop</id>
+    </provides>
+    <branding>
+        <color type="primary" scheme_preference="dark">#752a2a</color>
+        <color type="primary" scheme_preference="light">#752a2a</color>
+    </branding>
+    <content_rating type="oars-1.1"/>
+    <url type="homepage">https://github.com/catacombing/kumo</url>
+    <url type="bugtracker">https://github.com/catacombing/kumo/issues</url>
+    <url type="vcs-browser">https://github.com/catacombing/kumo</url>
+    <screenshots>
+        <screenshot type="default">
+            <image>https://raw.githubusercontent.com/catacombing/kumo/refs/heads/master/screenshots/main_ui.png</image>
+            <caption>Kumo with the pine64.org landing page loaded</caption>
+        </screenshot>
+        <screenshot>
+            <image>https://raw.githubusercontent.com/catacombing/kumo/refs/heads/master/screenshots/tab_groups.png</image>
+            <caption>Tabs organized in a tab group</caption>
+        </screenshot>
+    </screenshots>
+    <releases>
+        <release version="0.0.0" date="2025-04-11">
+            <description>
+                <p>Preview release of Kumo</p>
+            </description>
+        </release>
```

Comment:
I've seen pre-releases made for other Flathub packages, so I'd assume this should be fine?

Technically no version has been "officially" released yet, since building WebKit is obviously an undertaking that is too much for more users. So I'd rather get an early "test" version out to get some more testers and refine things more before marking an official "stable" release.

That said the browser is functional and my sole browser on my phone right now. So despite the lack of a release, I do consider this app ready to be used by a broader audience.